javascript-if/main.js

```diff
@@ -0,0 +1,94 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(number) {
+  let result;
+  if (number < 5) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function isEven(number) {
+  let result;
+  if (number % 2 === 0) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function startsWithJ(string) {
+  let result;
+  if (string.charAt(0) === 'J') {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function isOldEnoughToDrink(person) {
+  let result;
+  if (person.age < 21) {
+    result = false;
+  } else {
+    result = true;
+  }
+  return result;
+}
+
+function isOldEnoughToDrive(person) {
+  let result;
+  if (person.age > 16) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  let result;
+  if (person.age > 21) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
```

Comment:
This should always return `false`:

---

![image](https://user-images.githubusercontent.com/16143771/152250172-235cb215-e086-4f4b-bc6f-f51958e12ffb.png)